ossfuzz/compress_frame_fuzzer.c

```diff
@@ -19,8 +19,10 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
     FUZZ_dataProducer_t *producer = FUZZ_dataProducer_create(data, LZ4_compressBound(size));
     LZ4F_preferences_t const prefs = FUZZ_dataProducer_preferences(producer);
-    size_t const compressBound = LZ4F_compressFrameBound(size, &prefs);
-    size_t const dstCapacity = FUZZ_dataProducer_uint32(producer, 0, compressBound);
+    size_t const compressBound = LZ4F_compressFrameBound(FUZZ_dataProducer_remainingBytes(producer), &prefs);
+    size_t const dstCapacitySeed = FUZZ_dataProducer_uint32_seed(producer, 0, compressBound);
+    size_t const dstCapacity = FUZZ_dataProducer_uint32(dstCapacitySeed,
+        0, FUZZ_dataProducer_remainingBytes(producer));
```

Comment:
minor :
I would rather recommend to move `size = FUZZ_dataProducer_remainingBytes(producer);` up, and use `size` directly. It makes it very clear that `size` is the same value for `dstCapacity` determination and as argument of `LZ4F_compressFrame()`,
in contrast with `FUZZ_dataProducer_remainingBytes(producer)`, which is a function call, and may (or may not) provide 2 different results, or generate side effects, for 2 consecutive operations.